File: java/DuplicateNumber.java

import java.util.HashSet;

public class DuplicateNumber {
    public static void main(String[] args) {
        int [] arr = new int[]{1,1,2,3,55,55};
        getDupes(arr);
    }

    public static void getDupes(int [] nums){
        HashSet<Integer> resSet = new HashSet<Integer>();
        for(int i = 0; i<nums.length;i++){
            if(!resSet.add(nums[i])){
                System.out.println(" Dupe of " +nums[i] +" at pos : " +i);
            }
            else{
                resSet.add(nums[i]);
            }
        }
    }
}

File: js/duplicateNumber.js
function getDupes(nums) {
    for(let i =0; i<nums.length-1;i++){
        for(let j =i+1; j<nums.length;j++){
            if(nums[i] == nums[j]){
                console.log(` Dupe of ${nums[i]} at pos : ${j}`);
                return;
            }
        }
    }
}

getDupes([99,1,2,3,99,99,3]);